vochain/transaction/tokens_tx.go

```diff
@@ -30,6 +31,10 @@ func (t *TransactionHandler) SendTokensTxCheck(vtx *vochaintx.Tx) error {
 	if len(tx.To) == 0 {
 		return fmt.Errorf("invalid to address")
 	}
+	if bytes.Equal(tx.From, tx.To) {
```

Comment:
Actually I think the check should go to line 54 and compare txFromAddress.Bytes() and txToAddress.Bytes().

This is because an attacker could add extra bytes on tx.From or tx.To, which would make the current comparison not match. However, when the bytes are converted to Ethereum Addresses, the bytes are cropped, thus the resulting addres could be the same.

Reply:
Makes sense, I will add the change